### ISSUE TITLE
Update/remove incorrect divs in Facility locator search results

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#18386]
## Description
I recently found som `<div>`s that should be cleaned up in the Facility Locator search results listings:
- There are 2 empty `<div>`s with min-heights that are causing white space. These should be removed:
![Screenshot 2024-06-24 at 4 07 58 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/979ed65a-6a3b-40d6-b172-834b3805adfe)
- The content should be wrapped using `<p>` tags instead of a `<div>` or a `<span>`. `<div>`s and `<span>`s are catch all tags that don't typically have any style and so the DST didn't give them any default style. For this reason, when the browser font sizes are set to something other than the standard, text within these tags does not resize properly
   - Using the current markup with `<div>` and `<span>`:
 
![Screenshot 2024-06-24 at 1 40 27 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/b5113584-0266-409c-8f1b-a5f30c4e69b0)

   - Using `<p>` tags instead: 
![Screenshot 2024-06-24 at 1 41 22 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/304b736c-da5c-4951-89ef-f4ab86f96ae9)

## Acceptance Criteria
- [x] No empty div's are appearing in search results
- [x] Content is wrapped using `<p>` tags instead of more generic tags
- [x] Requires design review
- [x] Requires accessibility review

## Accessibility review steps
To confirm the font is resizing correctly:
1. Go to the Facility Locator page `/find-locations`
2. Complete a search so that the search results appear
3. Within your browser, update your settings to the largest font size available.
   - Chrome: Settings > Appearance > Font size
   - Firefox: Settings > Language & Appearance > Fonts
   - Edge: Settings > Appearance > Fonts
4. Confirm all the text within each of the search result items increases in size correctly

**Comment by eselkin** [comment]
There's a span around the facility's title... It has no style attached but an occasional click/keypress tracking on VA Health facilities. I can remove it and put the click on the h3 title, but not sure we want to take on more intricate stuff here in this ticket.

**Comment by laflannery** [comment]
@eselkin From an accessibility standpoint, this `<span>` isn't doing anything bad. The heading tag has the proper font settings so it's resizing and scaling properly.

I don't disagree with you that this tag should maybe not be here from a clean up standpoint though. This might be a product call - if we expand this ticket or create a new one? @Agile6MSkinner

**Comment by Agile6MSkinner** [comment]
I'd rather get it done in this ticket if it doesn't create any issues.

**Comment by eselkin** [comment]
So, I didn't see a response on how @laflannery wanted to handle this. I'll make an attempt to have this changed

**Comment by jilladams** [comment]
If you're blocked on the a11y feedback while Laura is out, we can also ask the A11y folks in #accessibility-champs, if that would be helpful.

**Comment by eselkin** [comment]
It all depends on whether they fix the mess that's going on with v-w presently. All unit tests are breaking now after some recent change they made and they even stopped the daily deploy.

**Comment by laflannery** [comment]
@eselkin I believe [my response above](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18386#issuecomment-2355790962) answer's your question. Is there something else that you need from me on this?

**Comment by laflannery** [comment]
Confirmed on prod!
